manifests/o/OpenTTD/OpenTTD/12.1/OpenTTD.OpenTTD.installer.yaml

```
@@ -0,0 +1,40 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: OpenTTD.OpenTTD
+PackageVersion: 12.1
+MinimumOSVersion: 7.0.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://cdn.openttd.org/openttd-releases/12.1/openttd-12.1-windows-win64.exe
+  InstallerSha256: B799DADF8E01469EF5A4FDB57927EACDE77FDFCBC95787EFC522B8C5B434071A
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://cdn.openttd.org/openttd-releases/12.1/openttd-12.1-windows-win32.exe
+  InstallerSha256: CA9918622A7F9CE34B5BB730E0B5DCC71D9D988E8D4F42BC45EC5210F0B1F920
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: arm64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://cdn.openttd.org/openttd-releases/12.1/openttd-12.1-windows-arm64.exe
+  InstallerSha256: 112F6B8E1FD7C624EF22A2F1C4EBD87C2134BF8BFBD9ECBBD8E8C2CC2B753F4B
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+  UpgradeBehavior: install

```

Instead of having the information repeated for every installer, it can be condensed down to the root level
```suggestion
InstallerType: nullsoft
Scope: machine
UpgradeBehavior: install
InstallerLocale: en-US
Installers:
- Architecture: x64
  InstallerUrl: https://cdn.openttd.org/openttd-releases/12.1/openttd-12.1-windows-win64.exe
  InstallerSha256: B799DADF8E01469EF5A4FDB57927EACDE77FDFCBC95787EFC522B8C5B434071A
- Architecture: x86
  InstallerUrl: https://cdn.openttd.org/openttd-releases/12.1/openttd-12.1-windows-win32.exe
  InstallerSha256: CA9918622A7F9CE34B5BB730E0B5DCC71D9D988E8D4F42BC45EC5210F0B1F920
- Architecture: arm64
  InstallerUrl: https://cdn.openttd.org/openttd-releases/12.1/openttd-12.1-windows-arm64.exe
  InstallerSha256: 112F6B8E1FD7C624EF22A2F1C4EBD87C2134BF8BFBD9ECBBD8E8C2CC2B753F4B
```

---

🤗It's clean. Thank!